### PR TITLE
new remove chair

### PR DESCRIPTION
**Pull request by DeltaNedas**
nuked it from orbit kek
and i fixed gradlew because my environment is the only environment
i use tabs and i wouldve replaced spaces with em but im a computer whiz and not js monkey trash

### CODE REVIEW

**Comment by Rubyjunk** [comment]
Your gradle edits ruin it for windows users... just one example: java8 is not on the windows path

**Comment by DeltaNedas** [comment]
gradle is for *nix not windows
inb4 reddit

File: gradlew [hunk 1]
```diff
@@ -81,15 +82,15 @@ Please set the JAVA_HOME variable in your environment to match the
 location of your Java installation."
     fi
 else
-    JAVACMD="java"
-    which java >/dev/null 2>&1 || die "ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
+    JAVACMD="java8"
```
Comment (DaMatrix) [inline gradlew]: uh... what? this is going to break everyone's environment except yours

Comment (DaMatrix) [reply gradlew]: just leave it as-is, nobody smart uses java >8

**Comment by DeltaNedas** [comment]
well if thats your only issue fine, ill revert it.

**Comment by DaMatrix** [comment]
just leave the gradle stuff as-is, it's worked fine for everyone except you for the past 3 years

**Comment by DeltaNedas** [comment]
ye read #254

**Pull request closed without merging**